### DISCUSSION TITLE
[Feature Request] Redraw background to new position after moving the window

### DISCUSSION THREAD

**Discussion by costis-t** [kovidgoyal/kitty#4780] [Ideas]
Is it possible to redraw the background when moving the kitty window to a new location ([example](https://imgur.com/a/ik8CAkz))?
It's like "fake" transparency: it hides the windows below kitty and shows the desktop wallpaper. Alternatively it could also use a background image (the same as the wallpaper) and change it's "cropping" margins.
![image](https://user-images.githubusercontent.com/20000896/156447678-b1d6915c-16f3-4094-be82-36a99ae1f993.png)

If there is a workaround already, please let me know. 
If not, is it possible to do this with a kitten?

Motivation:
There was a terminal which was doing this but I can't recall which one. Perhaps it wasn't even a terminal but a window manager (perhaps FVWM). It's a bit boring to look at the same location of a background image after many hours.

**Comment by kovidgoyal** [comment]
kitty has no such facility, but since you can control the background
image via remote control, you can always write a script to do it for you
and integrate it via your window manager assuming the window manager
provides some hooks for when a window is moved.
https://sw.kovidgoyal.net/kitty/remote-control/
